### PR TITLE
BZ#2176838: Updated VM SSH procedure

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.12+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2176838

Link to docs preview: https://60053--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-accessing-vm-consoles.html#virt-accessing-vmi-ssh_virt-accessing-vm-consoles

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by fabiand** [comment]
lgtm!

Thanks

File: modules/virt-accessing-vmi-ssh.adoc [hunk 1]
```diff
@@ -62,16 +63,17 @@ spec:
...
-. Restart the VM to apply your changes.
+.. Restart the VM to apply your changes.
 
-. Run the following command to access the VM via SSH:
+. Connect to the VM via SSH: 
+.. Run the following command to access the VM via SSH:
 +
 [source,terminal]
 ----
 $ virtctl ssh -i <key_file> <vm_username>@<vm_name>
```
[the remark this answers is not in the document]
Comment (sjhala-ccs) [reply modules/virt-accessing-vmi-ssh.adoc:73]: @fabiand What do you think?

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

File: modules/virt-accessing-vmi-ssh.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (fabiand) [reply modules/virt-accessing-vmi-ssh.adoc:73]: FWIW - I'd check what other cli examples in the docs do.

`-n` is a common flag.
We can for sure include it, but maybe it's considered common knowledge.

I'd throw the ball back and say: Check what other CLI snippets in the CNV docs do

**Pull request merged**